admin/src/components/NavBar.vue

```diff
@@ -33,8 +33,8 @@ export default {
   name: 'navbar',
   methods: {
     logout() {
+      window.location = CONFIG.WALLET_URL
```

Comment:
I am not sure that the dispatch is executed after changing the window location. Furthermore we use `window.location.assign` so we are able to test it. See in frontend `Layout/DashboardLayout_gdd.vue`